src/components/SimpleBSTOperations.js

```
@@ -0,0 +1,395 @@
+    });
+    
+    /**
+     * Helper function to find minimum value node in a subtree
+     */
+    const findMinValue = (node) => {
+      if (!node || !node.children) return null;
+      
+      let current = node;
+      
+      // Traverse left as far as possible to find minimum
+      while (current.children && current.children[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)